bpf/kprobe_pwru.c

```diff
@@ -348,7 +348,7 @@ set_output(struct pt_regs *ctx, struct sk_buff *skb, struct event_t *event, stru
 		set_tuple(skb, &event->tuple);
 
 	if (cfg->output_skb)
-		set_skb_btf(skb, &event->print_skb_id);
+		set_skb_btf(skb, (typeof(print_skb_id) *)(void *)&event->print_skb_id);
```

Comment:
I'd rather disable the warning explicitly using `-Wno-address-of-packed-member` to make it explicit we're silencing/ignoring it. With the current solution using the `void *` cast it's less obvious IMO.

Reply:
@tklauser done as suggested.